test/distributions/test_distributions.py

```diff
@@ -739,6 +761,16 @@ def is_all_nan(tensor):
             'concentration': torch.tensor([-1.0], requires_grad=True)
         }
     ]),
+    Example(Wishart, [
+        {
+            'covariance_matrix': torch.tensor([[1.0, 0.0], [0.0, -2.0]], requires_grad=True),
+            'df': torch.tensor([1.], requires_grad=True),
+        },
+        {
+            'covariance_matrix': torch.tensor([[1.0, 1.0], [1.0, -2.0]], requires_grad=True),
+            'df': torch.tensor([3.], requires_grad=True),
+        }
+    ]),
     Example(ContinuousBernoulli, [
         {'probs': torch.tensor([1.1, 0.2, 0.4], requires_grad=True)},
         {'probs': torch.tensor([-0.5], requires_grad=True)},
```

Comment:
Let us add a few more tests:
 - Testing against the scipy sampler. e.g. https://github.com/pytorch/pytorch/blob/master/test/distributions/test_distributions.py#L1512, https://github.com/pytorch/pytorch/blob/master/test/distributions/test_distributions.py#L1663
 - Do a grad check on log_prob.